notebooks/DrizzlePac/align_to_catalogs/align_to_catalogs.ipynb

```diff
@@ -4,42 +4,80 @@
    "cell_type": "markdown",
```

Comment:
<p>Thank you for your work. I would like to suggest that you use more descriptive names for your variables. This will enhance the readability.</p><p><code>obs_ids -&gt; observations_ids,</code></p><p><code>props -&gt; proposal_ids</code> ,</p><p><code>filts-&gt; filters</code> ,</p><p><code>obsTable-&gt; observation_table</code> ,</p><p><code>product -&gt; product_list</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/265/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
We absolutely appreciate the importance of clear and descriptive variable names. In following the discussion on the JIRA ticket, we would opt to forgo changing variable names. In this case, specific variables such as "obs_ids" are chosen to match the columns in the MAST query. In addition, words like "filter" call built-in python functions, which could cause errors. Please let us know if this is an acceptable compromise as your time allows. Thank you!